ViAn/Video/video_player.cpp

```diff
@@ -4,7 +4,9 @@
 #include "GUI/mainwindow.h"
 #include <thread>
 #include <QWaitCondition>
+#include <QDesktopWidget>
 #include <qpainter.h>
+#include <qdebug.h>

```

Comment:
Is `#include <qdebug.h>` needed? (I tried commenting it out and it was no prolem.)